app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,67 @@
+@import "bootstrap";
+@import "font-awesome";
+
+@font-face {
+  font-family: 'NotoSerif';
+  src: url('NotoSerif.ttf');
+}
+
+:root {
+  --bs-body-bg: #fffbfb;
+}
+
+.my_alert {
+  color: red;
+}
+
+.errors {
+  color: red;
+  border: 1px solid;
+  padding: 1em;
+}
+
+.banner-background {
+  font-family: 'NotoSerif';
+  background-image: url('head_back.jpg');
+  background-size: cover;
+}
+
+.banner-overlay {
+  background: rgb(0, 0, 0);
+  background: linear-gradient(344deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 100%);
+}
+
+label {
```

Comment:
Looks like you've put some extra work into styling some of the other elements, which I love to see!